erdjs/src/smartcontracts/typesystem/typeMapper.ts

```diff
@@ -39,9 +40,19 @@ export class TypeMapper {
             ["MultiResultVec", VariadicType],
             ["variadic", VariadicType],
             ["OptionalArg", OptionalType],
+            ["optional", OptionalType],
             ["OptionalResult", OptionalType],
             ["MultiArg", CompositeType],
             ["MultiResult", CompositeType],
+            // Perhaps we can adjust the ABI generator to only output "tuple", instead of "tupleN"?
```

Comment:
@andrei-marinica?

But it can remain `tupleN` for now, until we have the time to adjust ABI generation in the Rust framework.

Reply:
I think tupleN is a bit more expressive, since it is more obvious how many arguments it has.
In Rust it is impossible to have a type with variable number of generic arguments, so the distinction is necessary. If it is not desired in the ABI, it is extremely easy to change on Rust size.